README.md

```diff
@@ -1,5 +1,9 @@
 # Doto
 
+![](https://github.com/se701g2/Doto/workflows/doto-CI/badge.svg?event=push) 
+![](https://github.com/se701g2/Doto/workflows/doto-backend-deploy/badge.svg?event=push) 
+![](https://github.com/se701g2/Doto/workflows/doto-frontend-deploy/badge.svg?event=push)
```

Comment:
Good work @cwon880 . Just wondering if these files have been added as they aren't rendering for me.

Reply:
nvm. hadn't worked with badges before.